cmd/output/error.go

```diff
@@ -76,3 +80,54 @@ func PrintErrorInvalidOrganizationSlug(slug string) {
 func PrintErrorInvalidAppSlug(appSlug string) {
 	PrintError("Invalid app %q.", "Must contain only lower-case alphanumerical characters and dashes.", appSlug)
 }
+
+func PrintErrorAppNotFound(ai appident.AppIdentifier) {
+	PrintError(
+		"App not found",
+		"The app you are trying to delete \"%s/%s\" cannot be found. Did you specify the correct organization and app slug?",
+		ai.OrganizationSlug, ai.AppSlug,
+	)
+}
+
+func PrintErrorAccessDenied(ai appident.AppIdentifier) {
+	PrintError(
+		"Access denied.",
+		`Hint: You may have specified an organization name instead of an organization slug.
+Is the organization slug %q and the app slug %q correct?`,
+		ai.OrganizationSlug, ai.AppSlug,
+	)
+}
+
+func PrintAppError(err error, ai appident.AppIdentifier) {
+	switch {
+	case errors.Is(err, app.ErrAccesDenied):
+		PrintErrorAccessDenied(ai)
+	case errors.Is(err, app.ErrAppNotFound):
+		PrintErrorAppNotFound(ai)
+	default:
+		PrintErrorDetails("Error occurred deleting app.", err)
```

Comment:
This also seems too specific for the naming function 😄

Reply:
Not sure what you mean?

Reply:
Ah, good point.